include/listener.h

```diff
@@ -43,7 +43,7 @@ class Listener {
 
     Listener(const Address& address);
     void init(
-            size_t workers,
+            ssize_t workers,
```

Comment:
Alright so I'd rather cast the return type of `hardware_concurrency()` to unsigned. workers should definitely not be negative